messaging/src/main/java/org/axonframework/eventhandling/processors/subscribing/SubscribingEventProcessorConfiguration.java

```diff
@@ -56,17 +59,32 @@ public class SubscribingEventProcessorConfiguration extends EventProcessorConfig
     private EventProcessingStrategy processingStrategy = DirectEventProcessingStrategy.INSTANCE;
 
     /**
-     * Constructs a new {@link SubscribingEventProcessorConfiguration} with default values.
+     * Constructs a new {@code SubscribingEventProcessorConfiguration} with default values.
+     *
+     * @param configuration The configuration, used to retrieve global default values, like
+     *                      {@link MessageHandlerInterceptor MessageHandlerInterceptors}, from.
+     */
+    @Internal
+    public SubscribingEventProcessorConfiguration(@Nonnull Configuration configuration) {
+        super(configuration);
+    }
+
+    /**
+     * Constructs a new {@code SubscribingEventProcessorConfiguration}.
+     * <p>
+     * This configuration will not have any of the default {@link MessageHandlerInterceptor MessageHandlerInterceptors}
+     * for events. Please use {@link #SubscribingEventProcessorConfiguration(Configuration)} when those are desired.
      */
     public SubscribingEventProcessorConfiguration() {
```

Comment:
There I suggest to mark it as an `@Internal.

Reply:
Ah yes, sharp! Will do.